src/playback/view/mixerpanelmodel.cpp

```diff
@@ -167,16 +163,6 @@ void MixerPanelModel::loadItems()
     setupConnections();
 }
 
-void MixerPanelModel::onNotationChanged()
-{
-    auto instrumentTrackIdMap = controller()->instrumentTrackIdMap();
-    for (auto it = instrumentTrackIdMap.cbegin(); it != instrumentTrackIdMap.cend(); ++it) {
-        if (MixerChannelItem* item = findChannelItem(it->second)) {
-            item->loadSoloMuteState(controller()->trackSoloMuteState(it->first));
```

Comment:
where is the loadSoloMuteState called now?

Reply:
It's called when the the channelitem is built, and aux channels still use it.

I added this block in #20758 to ensure that solo states were loaded properly on notation changes - this is no longer necessary with the recent changes (now that `PlaybackController::updateSoloMuteStates` handles everything related to notation changes).